web/src/components/eventdetail/bookclub.js

```diff
@@ -0,0 +1,77 @@
+import {
+  Avatar,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import NonAvatar from '../../img/noavatar-profile.jpg';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+
+// TODO(YoonYeoHwan): Have to match event address and datetime,
+// because different event schedule has different address.
+// Have to talk about design.
```

Comment:
"have to talk about design" doesn't say what issues need to be discussed. either explain concretely or remove this line.